### ISSUE TITLE
custom assets crash.

### ISSUE THREAD

**Issue by casigus** [TheGameCreators/GameGuruRepo#4654]
imported some new assets, to build on a map. after adding the assets to the map GGMAX crashes every time testing.

After deleting the new imported modeels, the level runs fine again.

This bug happened before too just after AMD fix, theres defo someething going on with models/custom assets and the importing of them causing max to crash when using those new assets.

steps.

imported new model
placeed them on the map
click test.
takes forever to load
fans rev up.
loading bar gets to completetion.
ggmax crashes to desktop.

solution (not ideal)

delete the imported models, level loads fine.

**Comment by davetgc** [comment]
can you post or supply the assets? It sounds like an issue with the imported models.

**Comment by casigus** [comment]
yeah its defo the imported models with the latest updates! sure can. Ill get them zipped up and sent over.

**Comment by casigus** [comment]
okies. discord the zip to you

**Comment by davetgc** [comment]
Requested OP forward files to @LeeBamberTGC for review.
